packages/react-components/src/icons/src/MultiVariantIcon.tsx

```diff
@@ -0,0 +1,81 @@
+import { ComponentProps, ElementType, ForwardedRef, ReactElement } from "react";
+import { Icon } from "./Icon";
+import { OrbitComponent, forwardRef, slot } from "../../shared";
+
+export interface CreatedMultiVariantIconProps {
```

Comment:
An instance of a React type is called a React element, therefore we could instead call this `MultiVariantIconElement` ?

Reply:
This is not a react type, nor a react element. It's an interface for the MultiVariantIconElement's props

Reply:
i renamed it to MultiVariantIconElementProps